spec/exhibits-solr-conf/configure_solr_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'rake'
+require 'hurley'
+require 'jettywrapper'
+
+describe 'exhibits namespace rake task' do
+  before do
+    load File.expand_path('../../../lib/exhibits-solr-conf/tasks/configure_solr.rake', __FILE__)
+  end
+
+  describe 'configure_solr' do
+    it 'takes argument for the target solr conf dir' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.